util/typo.php

```diff
@@ -1,59 +1,67 @@
 <?php
-	// Tired of chasing typos and finding them after a commit.
-	// Run this from cmdline in basedir and quickly see if we've
-	// got any parse errors in our application files.
 
+// Tired of chasing typos and finding them after a commit.
+// Run this from cmdline in basedir and quickly see if we've
+// got any parse errors in our application files.
 
-	error_reporting(E_ERROR | E_WARNING | E_PARSE );
-	ini_set('display_errors', '1');
-	ini_set('log_errors','0');
+use Friendica\App;
 
-	include 'boot.php';
+error_reporting(E_ERROR | E_WARNING | E_PARSE);
+ini_set('display_errors', '1');
+ini_set('log_errors', '0');
 
-	$a = new App();
+include 'boot.php';
 
-	if(x($a->config,'php_path'))
-		$phpath = $a->config['php_path'];
-	else
-		$phpath = 'php';
+$a = new App(dirname(__DIR__));
 
+if (x($a->config, 'php_path')) {
+	$phpath = $a->config['php_path'];
+} else {
+	$phpath = 'php';
+}
 
-	echo "Directory: mod\n";
-	$files = glob('mod/*.php');
-	foreach($files as $file) {
-        passthru("$phpath -l $file", $ret); $ret===0 or die();
-	}
+echo "Directory: mod\n";
+$files = glob('mod/*.php');
+foreach ($files as $file) {
+	passthru("$phpath -l $file", $ret);
+	$ret === 0 or die();
+}
 
-	echo "Directory: include\n";
-	$files = glob('include/*.php');
-	foreach($files as $file) {
-        passthru("$phpath -l $file", $ret); $ret===0 or die();
-	}
+echo "Directory: include\n";
+$files = glob('include/*.php');
+foreach ($files as $file) {
+	passthru("$phpath -l $file", $ret);
+	$ret === 0 or die();
+}
 
-    echo "Directory: object\n";
-	$files = glob('object/*.php');
-	foreach($files as $file) {
-        passthru("$phpath -l $file", $ret); $ret===0 or die();
-	}
+echo "Directory: object\n";
+$files = glob('object/*.php');
+foreach ($files as $file) {
+	passthru("$phpath -l $file", $ret);
+	$ret === 0 or die();
+}
 
```

Comment:
Yes it is. This is open because I didn't change the line @annando added his comment on.